docs/spec/v1beta2/alerts.md

```diff
@@ -162,6 +162,13 @@ starting the controller with the `--no-cross-namespace-refs=true` flag.
 When this flag is set, alerts can only refer to event sources in the same namespace as the alert object,
 preventing tenants from subscribing to another tenant's events.
 
+### Event metadata
+
+`.spec.eventMetadata` is an optional field for adding metadata to events emitted by the
+controller. Metadata fields added by the controller have priority over the fields
+added here, and the fields added here have priority over fields originally present
+in the event.
+
```

Comment:
Please add here an example section with metadata for:
```yaml
app.kubernetes.io/env : "production"
app.kubernetes.io/cluster: "my-cluster"
app.kubernetes.io/region: "us-east-1"
```

Reply:
done!